### PR TITLE
use `objectCount()` from ospsuite.utils

### PR DESCRIPTION
**Pull request by IndrajeetPatil**
And remove the internal copy

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -42,7 +42,7 @@ build_script:
   - travis-tool.sh install_deps
   - travis-tool.sh r_binary_install curl
   - Rscript -e "install.packages('covr', repos = 'http://cran.us.r-project.org')"
-  - Rscript -e "install.packages('ospsuite.utils', repos = 'http://cran.us.r-project.org', type = 'source')"
+  - Rscript -e "install.packages('https://ci.appveyor.com/api/buildjobs/sdy8a68mbyxpctx2/artifacts/ospsuite.utils_1.2.0.9000.zip', repos = NULL, type = 'binary')"
```
Comment (PavelBal) [inline appveyor.yml:45]: Always use the latest version?

Comment (msevestre) [reply appveyor.yml:45]: This needs to change tto use the latest version

**Review by msevestre — CHANGES_REQUESTED** [review]

File: appveyor.yml (hunk 1 above)
Comment (IndrajeetPatil) [reply appveyor.yml:45]: The latest commit in _ospsuite.utils_ didn't work:
https://ci.appveyor.com/project/open-systems-pharmacology-ci/ospsuite-rutils/builds/43044129/artifacts

There is nothing at 
https://ci.appveyor.com/api/projects/open-systems-pharmacology-ci/ospsuite-rutils/artifacts/ospsuite.utils.zip

**Comment by IndrajeetPatil** [comment]
This will be part of PR to close #853

**Pull request closed without merging**